components/symbol/tests/unit/symbol.spec.tsx

```diff
@@ -0,0 +1,83 @@
+import { GEL } from '@westpac/core';
+import { render, screen } from '@testing-library/react';
+import wbc from '@westpac/wbc';
+import { overridesTest } from '../../../../helpers/tests/overrides-test.js';
+import { nestingTest } from '../../../../helpers/tests/nesting-test.js';
+import * as symbols from '@westpac/symbol';
+import { Symbol } from '../../src/Symbol.js';
+import { blenderSymbol } from '../../src/overrides/symbol.js';
+
+overridesTest({
+	name: 'symbol',
+	overrides: ['Symbol'],
+	Component: (props: any) => <Symbol copyrightYear="2020" align="center" {...props} />,
+});
+
+nestingTest({
+	name: 'symbol',
+	Component: (props: any) => <Symbol {...props} />,
+});
+
+// using any typing on props as symbol is not TS and does not have an interface
+function SimpleSymbol(props: any) {
```

Comment:
Just so it's extra clear, like icon, symbol is not a TS file so it has no type interface to use which is why any is used here. Could be updated in the future if the symbol component gets converted to TS.